sdk/appconfiguration/app-configuration/src/models.ts

```diff
@@ -292,10 +292,25 @@ export interface ListSettingsOptions extends OptionalFields {
  */
 export interface ListConfigurationSettingsOptions extends OperationOptions, ListSettingsOptions {}
 
+/**
+ * @interface
+ * An interface that tracks the settings for paged iteration
+ */
+export interface PageSettings {
+  /**
+   * @member {string} [continuationToken] The token that keeps track of where to continue the iterator
+   */
+  continuationToken?: string;
+  // The appconfig service doesn't currently support letting you select a page size
+  // so we're ignoring their setting for now.
```

Comment:
Nit: This doesn't show up in docs does it?

Reply:
Nope, doesn't.